### PR TITLE
Updating conftest to progress with testing

### PR DESCRIPTION
**Pull request by koshy-thomas**
## Description

Updating conftest to progress with completing/correcting tests with the current APIs. I have removed some code which is not needed and added some to make it work.

We can start running tests individually after this and start correcting/adding and covering all the existing api methods
commands to be used:
cd pyC8/tests
pytest -v filename

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

* C8 Version:

## Reviews

Please identify two developers to review this change

- [ ] @ricardo-macrometa 
- [ ] @dlozina-macrometa 
- [ ] @edgargarciamacrometa 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added new functional tests that prove my fix is effective or that my feature works
- [ ] Existing and new functional tests pass with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by koshy-thomas** [comment]
Updated helpers.py to avoid errors to length of string and added chars to generate_string to avoid errors related to validation of Password

**Comment by koshy-thomas** [comment]
Updated fabric.py with correct implementation of create_fabric api and added / to _fabric endpoints so as to avoid errors in connection.py when the fabric name also contains _fabric and enpoint also contains _fabric

**Comment by koshy-thomas** [comment]
Updated connection.py with the above logic to check for /_fabric instead of just _fabric

**Comment by koshy-thomas** [comment]
Updated Tenant.py with displayName and email in response to let the user view these relatable info as well and also so that the user maybe able to do more operations based on email (which is already known and unique) rather than username(UUID) which would be generated by the system.

**Comment by koshy-thomas** [comment]
And finally updated conftest.py with a working base so as to enable us to fix the individual test file now

File: c8/connection.py [hunk 1]
```diff
@@ -177,9 +177,8 @@ def send_request(self, request, customPrefix=None):
         # Below line is a debug to show what the full request URL is.
         # Useful in testing multitenancy API calls
         # if '_tenant' in request.endpoint and '_fabric' in request.endpoint:
-        if '_fabric' in request.endpoint:
+        if '/_fabric' in request.endpoint:
```
Comment (dlozina-macrometa) [inline c8/connection.py:180]: Nice catch here @koshy-thomas 😊

**Review by dlozina-macrometa — APPROVED** [review]
Great work 👍

**Review by ricardo-macrometa — APPROVED** [review]

**Pull request merged**